src/index.js

```diff
@@ -1,5 +1,5 @@
 import h from "./h"
 import app from "./app"
-import router from "./router"
+import Router from "./router"
```

Comment:
Exactly. I think Plugins written in title case would look better, just like Components.

```jsx
app({ plugins: [Router, Logger, Debugger, LevelDB, URLParser, Portals] })
```